### ISSUE TITLE
Reuse depiction string when asking categories

### ISSUE THREAD

**Issue by jidanni** [commons-app/apps-android-commons#4033]
Okay, so there are two (main) questions the app is asking. One is what does your picture depict, another is what categories does your picture belong to.

Therefore if I enter 'Bubulinos parthiogenesis' to answer the first question, wouldn't it be great to have that copied automatically into the second question to help me get started picking the categories instead of me having to type it in all over again or copy and paste?

Sure, don't one hundred percent force me to use it right away,  but have it nearby with a checkmark or something if I need it, to help me  get started picking those categories.

2.13.1.828~2ed5a74

**Comment by jidanni** [comment]
In fact even straight from the title, and the description, have that same string follow us around in case we need it. (Or you could say if we were smart we would know to copy the title ourselves while we're entering it and now we could use it just like I said.)

**Comment by nicolas-raoul** [comment]
Great idea!
Actually if you search for 'Bubulinos parthiogenesis' and 'apomixis' in depictions, my opinion is that in the category dialog we should display the top 3 results for both of these search terms.
The difficulty is to decide what constitutes a search term. For instance, if you want to search for the "dog" depiction and mistype "dig", then correct, now you have these search terms: "d" "di" "dig" "do" "dog".

By the way, we already suggest categories that are associated with the selected Wikidata items (depictions).

**Comment by Saral-code** [comment]
I think we can display top 3 result for categories as per depiction terms as suggested by @nicolas-raoul 
and if no categories are found we can keep it empty. 
I would like to work on this issue.

**Comment by nicolas-raoul** [comment]
@Saral-code Thanks for your enthusiasm! Would you mind focusing on bugs first, though? See bullet 3 of https://github.com/commons-app/commons-app-documentation/blob/master/android/Volunteers-welcome!.md#developers :-)

**Comment by Saral-code** [comment]
> @Saral-code Thanks for your enthusiasm! Would you mind focusing on bugs first, though? See bullet 3 of https://github.com/commons-app/commons-app-documentation/blob/master/android/Volunteers-welcome!.md#developers :-)

OK I'll be back here after 5 PRs. 👍

**Comment by nicolas-raoul** [comment]
@Saral-code Wonderful! :-)

**Comment by Ayan-10** [comment]
<img src="https://user-images.githubusercontent.com/71203077/123534983-9a346000-d73e-11eb-8b06-e5b52644c595.png" width=300/>

<img src="https://user-images.githubusercontent.com/71203077/123535016-e1225580-d73e-11eb-937f-0db0963b3939.png" width=300/>

@nicolas-raoul @jidanni As you can see in the images. The app is recommending selected depicts in categories. So can you elaborate on what the issue is about?

**Comment by nicolas-raoul** [comment]
@Ayan-10 You are right for the Tuesday example, we are very fortunate that it works in the majority of cases. Unfortunately, it sometimes does not work that well, sometimes because the item is not yet linked to the category, sometimes because there is no direct 1-1 relationship between items and categories.
